plugins/modules/ipapermission.py

```diff
@@ -371,6 +371,10 @@ def main():
                     for _member, _member_change in check_members.items():
                         if _member_change is not None:
                             _res_list = res_find[_member]
+                            if isinstance(_res_list, str):
```

Comment:
Is this needed for client context? Please add a comment.

Reply:
It is needed for the client context, at least, part of it. I'll add a comment.

Reply:
fixed.